docs/Migrations.md

```diff
@@ -0,0 +1,22 @@
+# Diesel Migrations
```

Comment:
Would be good to add some high level text to describe that this doc is about how to properly manage DB schema changes, etc etc.   Maybe change the heading to reflect that as well..